### ISSUE TITLE
call_deferred() and set_deferred() never raise error messages when invalid (as opposed to call)

### ISSUE THREAD

**Issue by chucklepie** [godotengine/godot#42680]
**Godot version:**
3.2.2

**OS/device including version:**
linux mint

**Issue description:**
If you call a method using call you get a trappable error and a message. If you use deferred then you get nothing, no stop and not even a debug message telling you the method/property does not exist. This will and can lead to difficult to debug faults. In addition whether you wrap a deferred around a method call or not, they should both have the same result, but they don't.

```
	call_deferred("blah",true) #no error or debug message
	set_deferred("blah",true) #no error or debug message
	call("blah") #stops code
```
I appreciate calling deferred is from some asynchronous step, but it should at least send a message to the error/log window.

**Steps to reproduce:**
Create code as above and run. One is trapped, the other goes silently.

**Comment by Xrayez** [comment]
This is strange because there's a merged PR which aimed to solve exactly this issue: #40485.

I confirm on 3.2.2 with the following MRP:
[godot_2020_10_10.00_33_46.zip](https://github.com/godotengine/godot/files/5357668/godot_2020_10_10.00_33_46.zip)

**Comment by chucklepie** [comment]
This is my version. I'm unsure from that defect whether it's cherry picked for 3.2.3 or in an earlier build (I asked this at the bug just in case this gets closed) :)

![image](https://user-images.githubusercontent.com/12863685/95634715-63b89700-0a82-11eb-81ed-6f9c4e047a51.png)

**Comment by chucklepie** [comment]
btw, i did search for this but when you enter call_deferred in the search bar that bug does not appear (mine does though).

**Comment by akien-mga** [comment]
> This is strange because there's a merged PR which aimed to solve exactly this issue: #40485.

Yes, and it was merged and cherry-picked for **3.2.3**.

Duplicate of #40401.
